src/patternfly/components/ContextSelector/context-selector.scss

```diff
@@ -18,6 +18,13 @@
   --pf-c-context-selector__toggle--active--BorderBottomColor: var(--pf-global--active-color--100);
   --pf-c-context-selector__toggle--expanded--BorderBottomWidth: var(--pf-global--BorderWidth--md);
   --pf-c-context-selector__toggle--expanded--BorderBottomColor: var(--pf-global--active-color--100);
+  --pf-c-context-selector__toggle--m-plain--Color: var(--pf-global--Color--200);
+  --pf-c-context-selector__toggle--m-plain--hover--Color: var(--pf-global--Color--100);
+  --pf-c-context-selector__toggle--m-plain--disabled--Color: var(--pf-global--disabled-color--200);
```

Comment:
Looks like you removed the block that uses this var, but I think we'll still want that for the `.pf-m-plain:not(.pf-m-text)` variation, since the plain/icon/kebab variation has a different text/icon color when disabled.